examp/workspace.sorted.toml

```diff
@@ -0,0 +1,8 @@
+[workspace]
+
+members = [
+    "1",
+    "2",
+    "3",
+    "4"
```

Comment:
Okay so this actually shows that there's still a regression compared to the latest crates.io release: This line should end in a comma. I don't mind much whether the logic is

- multi-line arrays always use a trailing comma or
- trailing commas in multi-line arrays are left alone

but we should definitely not be erasing such a trailing comma.